app/GenreAnime.php

```diff
@@ -4,13 +4,16 @@
 
 use Jenssegers\Mongodb\Eloquent\Model;
 use Jikan\Request\Genre\AnimeGenresRequest;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
 
 /**
  * Class Magazine
  * @package App
  */
 class GenreAnime extends JikanApiSearchableModel
 {
+    use HasFactory;
+
     protected array $filters = ["order_by", "sort"];
```

Comment:
What does the `filters` here signify for `GenreAnime` and `GenreManga`?

Currently, we don't have `order_by` or `sort` for these endpoints. But we do have [filter](https://docs.api.jikan.moe/#tag/genres/operation/getAnimeGenres)

Reply:
This should be set to an empty array.

Reply:
👍 done